backend/crud.py

```diff
@@ -39,6 +57,30 @@ def create_attribute(db: Session, data: AttributeCreateSchema, commit: bool = Tr
     return a
 
 
+def get_schemas(db: Session, all: bool = False, deleted_only: bool = False) -> List[Schema]:
+    q = select(Schema)
+    if all:
+        pass
+    elif deleted_only:
+        q = q.where(Schema.deleted == True)
+    else:
+        q = q.where(Schema.deleted == False)
```

Comment:
I believe you can simplify this block a little bit to something like this:

```python
if not all:
    q = q.where(Schema.deleted == deleted_only)
```

Reply:
Oh, sorry, I applied your change in another function which had the same lines 😅 I'll change this one too